src/routes/settings/settings.tsx

```diff
@@ -0,0 +1,22 @@
+import { Button } from "../../components"
+
+export const Settings: React.FC = () => {
+  return (
+    <div className="flex flex-col bg-lightGray items-center gap-8">
```

Comment:
This div is unnecessary. You can remove it completely and the closing tag on line 20.